build/cqs.tsv

```diff
@@ -126,7 +126,7 @@ org.apache.avro:avro	1.7.5	compile
 org.apache.camel:camel-core	2.24.1	compile
 org.apache.camel:camel-scala	2.24.1	compile
 org.apache.commons:commons-collections4	4.4	compile
-org.apache.commons:commons-compress	1.18	compile
+org.apache.commons:commons-compress	1.20	compile
```

Comment:
existing cq here, but not yet approved: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=21771

Reply:
Approved!